tests/unit/CurrencySymbolUtilsTest.js

```diff
@@ -0,0 +1,34 @@
+import _ from 'underscore';
+import * as CurrencySymbolUtils from '../../src/libs/CurrencySymbolUtils';
+
+// Taken from GetCurrencyList API.
+import currencyList from './currencyList.json';
```

Comment:
Could you maybe instead make this comment a bit longer explaining this file needs to be kept up to date and how to update it if need be? You could leave a link there in the comment if there is one which explains this well.

Reply:
Is this good?
```
// This file can get outdated. You can follow these steps to update it.
// src/libs/API.js
// GetCurrencyList().then(data => console.log(data.currencyList));
// it will output the json in console, you can copy it and use any external json formatter
// finally update currencyList.json
```

Reply:
Looks good! I have made couple of suggestions to make it may be easier to read.
```
// This file can get outdated. In that case, you can follow these steps to update it:
// - in src/libs/API.js
// - call: GetCurrencyList().then(data => console.log(data.currencyList));
// - copy the json from console and format it to valid json using some external tool
// - update currencyList.json
```

Reply:
Thanks @mountiny